app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationCancellationAPI10.scala.txt

```diff
@@ -0,0 +1,15 @@
+@(params: Map[String, Any])
+Alcohol Wholesaler Registration Scheme (AWRS) cancellation
+
+Your request to cancel your AWRS registration for @{params("organisationName")} was received on @{params("submissionDate")}.
+
+What happens next
+
+Your details on the AWRS register will be changed to show that your business has deregistered on the @{params("submissionDate")}.
+
+Unless you are joining a group or partnership, you must not trade alcohol wholesale after the @{params("submissionDate")}.
+
+You must sell or dispose of any existing wholesale stock by this date. If you continue to sell alcohol wholesale without an AWRS number you could be liable to criminal prosecution, financial penalties and/or seizure of alcohol stock (whether or not duty paid).
+
+
```

Comment:
Please add the signature to the txt versions too.
EG: https://github.com/hmrc/hmrc-email-renderer/blob/master/app/uk/gov/hmrc/hmrcemailrenderer/templates/awrs/awrsNotificationConfirmationAPI4.scala.txt#L21